src/main/java/com/rackspace/ceres/app/services/QueryService.java

```diff
@@ -123,7 +129,44 @@ public QueryService(ReactiveCqlTemplate cqlTemplate,
         ))
         .checkpoint();
   }
+  
+  public Flux<TsdbQueryResult> queryTsdbDownsampled(String tenant, List<TsdbQueryRequest> queries, Instant start, Instant end,
+      List<Granularity> granularities) {
+    return metadataService.getMetricsAndTagsAndMetadata(queries, granularities)
```

Comment:
This falls in line with how we are handling flux in our other API's so its fine. But it feels wrong. 

The API service should have a clear thing that it is doing, which is delegating responsibility to some other @Service and only handling API related things. Most of this data manipulation feels like it should be happening somewhere else, and stringing along all of these lambdas like this make it difficult to get whats going on at a glance. Mostly this comment is to start a larger discussion about this and maybe reconsider this style that we are using (maybe this is just the flux way).